starter_code/src/movies.js

```diff
@@ -15,6 +54,49 @@
 
 
 // Order by title and print the first 20 titles
+var moviesTitle = 
+movies.sort(function (a, b){
+    if (a.title > b.title){
+        return -1;
+    }
+
+    if (a.title < b.title){
+        return 1;
+    }
+
+    return 0;
+});
+
+moviesTitle.push(movies.reverse());
+
+moviesTitle.splice(20, moviesTitle.length);
+
+console.log(moviesTitle);
+
+var first20 = [];
+function orderAlphabetically(oneMovie){
+  oneMovie.forEach(function (oneMovie){
+  first20.push(oneMovie.title);
+});
+  console.log(first20);
+};
+
```

Comment:
Again, you should put every thing in one generic version. Also, if you want the sort method to sort thing in the opposite order you just have to switch 1 and -1
Here is the most simple way to do it : 
```js
function orderByDuration(moviesArray) {
  moviesArray.sort(function(a, b) {
    if (a.duration === b.duration) {
      if (a.title > b.title) {
        return 1;
      } else return -1;
    }
    return a.duration - b.duration;
  });
  return moviesArray;
}
```